apps/angular/src/core/dtos/backendEnums/animeStatusEnum.ts

```diff
@@ -0,0 +1,8 @@
+// Because we get this enum type from backend
+/* eslint-disable @typescript-eslint/naming-convention */
+/** Represents list of anime statuses. */
+export enum AnimeStatusEnum {
+	AIRING,
+	FINISHED,
+	NOT_YET_AIRED,
```

Comment:
We use snake case only for constants, not enums. Did you mean that:
```suggestion
	NotYetAired = 'NOT_YET_AIRED',
```